src/components/styles.js

```diff
@@ -23,7 +23,7 @@ export const ClauseHeader = styled.div`
   font-family: ${props => props.headerfont || 'Graphik'};
   grid-area: two / two / ten / ten;
   transition-duration: 0.5s;
-  background: ${props => props.headerbg || 'linear-gradient(180deg, #FFFFFF 0%, #ECF0FA 100%)'};
+  background: linear-gradient(180deg, #FFFFFF 0%, ${props => props.headerbg || '#ECF0FA'} 100%);
```

Comment:
does this work written like this? i would have expected it to look like `background: ${props => (`linear-gradient(180deg, #FFFFFF 0%, ${props.headerbg || '#ECF0FA'} 100%)`)};`

Reply:
Yeah it works the way I have it and the way you just put, but the way I have it is just shorter.